biz.aQute.bndlib/src/aQute/bnd/plugin/jpms/JPMSModuleInfoPlugin.java

```diff
@@ -85,28 +86,32 @@
 		SYNTHETIC(ACC_SYNTHETIC),
 		MANDATED(ACC_MANDATED);
 
-		public static Access parse(String input) {
-			switch (input) {
-				case "OPEN" :
+		public static Set<Access> parse(String input) {
+			switch (input.toLowerCase()) {
```

Comment:
```suggestion
			switch (input.toLowerCase(Locale.ROOT)) {
```

Plain `toLowerCase()` is local sensitive and may mess up in certain locales. See the javadoc for `toLowerCase()`.

Reply:
fixed